OSD-2022.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: OSD-2022
+title: Open Source Day 2022
+permalink: /OSD-2022/
+---
+:page-liquid:
+
+
+Welcome Open Source Day participants! We're glad to have you join us for the hackathon.
+
+To get started, follow the steps below:
+
+. *Setup the necessary tools to get started.* We use Java 11, Maven, and GitHub for development. Follow the setup instructions for your operating system:
++
+++++
+{% include OSD-2022-logo-bullets.html %}
+++++
+
+. *Pick your https://en.wikipedia.org/wiki/Comparison_of_integrated_development_environments#Java[IDE].* If it's your first time, we recommend an IDE which includes all the tools you'll need, like https://www.jetbrains.com/idea/[IntelliJ IDEA] or https://www.eclipse.org/ide/[Eclipse]. For those more familiar with Java development and using Maven, feel free to use whatever you'd like.
+. *Get familiar with WildFly.* Our link:../getting-started-for-developers/[Getting Started] page has some resources to help you learn more about the project. You don't need to go through all of the links, so here are a few sections to look at:
```

Comment:
s/WildFly/WildFly Elytron

s/You don't need to go through all of the links, so here are a few sections to look at/Want to get started quickly? Check out these sections: